app/models/position_history.rb

```diff
@@ -12,7 +12,7 @@ def last_position_finished
     return if employee_position_histories.blank?
 
     if employee_position_histories.order(id: :desc).take(1).pluck(:finished_on).first.nil?
-      errors.add(:base, "The last position is active")
+      errors.add(:base, I18n.t('activerecord.errors.models.position_history.base.last_position_active'))
```

Comment:
here and all other custom validations should be just:
`errors.add(:base, :last_position_active)`